CaT/src/Filter/FilterFactory.php

```diff
@@ -195,10 +226,10 @@ public function dateperiod_overlaps_or_empty_predicate($field_start, $field_end)
 	 * Map this over a text filter to get the standard behaviour where
 	 * a field is compared with the filter.
 	 *
-	 * @param	string	$field
+	 * @param	string	$field_id
 	 * @return	\Closure
 	 */
-	public function text_equals($field) {
+	public function text_equals($field_id) {
```

Comment:
Fall back to text_equals for dedup. This is merely a helper, as i did not expect fields to be created somewhere else, at is it now.